Python/PRP/Avatar/pyClothingItem.cpp

```diff
@@ -44,30 +45,152 @@ PY_METHOD_VA(ClothingItem, setMesh,
     pyKey* key;
 
     if (!PyArg_ParseTuple(args, "iO", &lod, &key)) {
-        PyErr_SetString(PyExc_TypeError, "read expects int, plKey");
+        PyErr_SetString(PyExc_TypeError, "setMesh expects int, plKey");
         return NULL;
     }
     if (!pyKey_Check((PyObject*)key)) {
-        PyErr_SetString(PyExc_TypeError, "read expects hsStream, plResManager");
+        PyErr_SetString(PyExc_TypeError, "setMesh expects plKey");
         return NULL;
     }
 
     self->fThis->setMesh(lod, *(key->fThis));
     Py_RETURN_NONE;
 }
 
+PY_METHOD_VA(ClothingItem, getElementTexture,
+    "Params: element, layer\n"
+    "Gets the Key of the texture for the specified element and layer")
+{
+    int element = 0;
+    int layer = plClothingItem::kLayerBase;
```

Comment:
It's not strictly necessary to set a default for these, since the parameters are not optional (and they won't be used uninitialized due to the early return).  But it's fine to leave them too.